packages/dashboard/src/app/views/editorSettings/videoCache/index.js

```diff
@@ -74,10 +74,6 @@ export default function VideoCacheSettings({
 }
 
 VideoCacheSettings.propTypes = {
-  isEnabled: PropTypes.bool.isRequired,
+  isEnabled: PropTypes.bool,
   updateSettings: PropTypes.func.isRequired,
 };
-
-VideoCacheSettings.defaultProps = {
-  isEnabled: false,
-};
```

Comment:
Is this change related?

Reply:
Not related; I was seeing these prop types warning during development and updated it here while I was at it.

`isEnabled` has a default value of `false` in the function signature, so it doesn't neeed `isRequired` nor `defaultProps`.